pkgs/tools/networking/norouter/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "norouter";
+  version = "0.6.4";
+
+  src = fetchFromGitHub {
+    owner = "norouter";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0h5jzxm4fw50781zj76r5ksnxkzsnrygrykpa913v9nd24c09c7m";
+  };
+
+  vendorSha256 = "sha256-DZ2kcNV8AzNogAUTaeus4rz9gCFo0wm306jcz/cAj0M=";
+
+  subPackages = [ "cmd/norouter" ];
+  doInstallCheck = true;
+  installCheckPhase = ''
+    $out/bin/norouter --version | grep ${version} > /dev/null
+  '';
+
+  meta = with lib; {
+    description = "NoRouter: IP-over-Stdio. The easiest multi-host & multi-cloud networking ever. No root privilege is required. ";
```

E.g., something like that:

```suggestion
    description = "Tool to handle unprivileged networking by using multiple loopback addresses";
```

see https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes